synapse/handlers/room.py

```diff
@@ -825,19 +828,26 @@ def send(etype, content, **kwargs):
                     # This will be reudundant on pre-MSC2260 rooms, since the
                     # aliases event is special-cased.
                     EventTypes.Aliases: 0,
+                    EventTypes.Tombstone: 100,
+                    EventTypes.ServerACL: 100,
                 },
                 "events_default": 0,
                 "state_default": 50,
                 "ban": 50,
                 "kick": 50,
                 "redact": 50,
-                "invite": 0,
+                "invite": 50,
             }
 
             if config["original_invitees_have_ops"]:
                 for invitee in invite_list:
                     power_level_content["users"][invitee] = 100
 
+            # Allow power levels to be defined per chat preset
+            preset_power_level_override = config.get("power_level_content_override")
+            if preset_power_level_override:
+                power_level_content.update(preset_power_level_override)
```

Comment:
bit of a danger of doing this like this is that when we want to override the PL for a single event type, the obvious approach won't work. I think it's ok though.